lib/livebook_web/live/settings_live.ex

```diff
@@ -60,6 +62,21 @@ defmodule LivebookWeb.SettingsLive do
               <% end %>
             </div>
           </div>
+          <!-- Autosave path configuration -->
+          <div class="flex flex-col space-y-4">
+            <div>
+              <h2 class="text-xl text-gray-800 font-semibold">
+                Auto save path
```

Comment:
Nitpick: according to wikipedia it is one word. :) And let's use location as a less technical term, wdyt?

```suggestion
                Autosave location
```